docker/docker-entrypoint.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
The entrypoint for this docker image needs to run the lambda handler. 

https://github.com/podaac/bignbit/blob/develop/docker/Dockerfile#L50-L51
https://github.com/podaac/bignbit/blob/develop/docker/docker-entrypoint.sh